api/v1alpha1/lvmcluster_types.go

```diff
@@ -90,6 +90,10 @@ type DeviceSelector struct {
 	// MinSize is the minimum size of the device which needs to be included. Defaults to `1Gi` if empty
 	// +optional
 	// MinSize *resource.Quantity `json:"minSize,omitempty"`
+
+	// Paths are the complete location of block devices, which needs to be included while creating a VG.
```

Comment:
// A list of device paths which would be chosen creating Volume Group.

Reply:
The comment should also discourage from using device names. Also, it would be good idea to include examples of the device paths ( like `/dev/disk/by-path/... ` etc) in the comment.

Reply:
changed the description and added the example as well